packages/shared/src/components/onboarding/FilterOnboarding.tsx

```diff
@@ -8,11 +8,13 @@ import { Origin } from '../../lib/analytics';
 interface FilterOnboardingProps {
   onSelectedTopics?(tags: Record<string, boolean>): void;
   className?: string;
+  setHasCategories?: (hasCategories: boolean) => void;
 }
 
 export function FilterOnboarding({
   onSelectedTopics,
   className,
+  setHasCategories,
```

Comment:
I would make this something more generic.. eg. `onSelectTopic` and then parent component can calculate `hasCategories` from topics it receives. It would be more reusable down the line if we want to hook up something else related to topics selection. WDYT?